### ISSUE TITLE
Accessibility:  Keyboard focus is lost in improved Statistics interface when selecting/unselecting checkboxes

### ISSUE THREAD

**Issue by claycompton** [NuGet/NuGetGallery#1067]
1.  Navigate to https://qa.nugettest.org/stats/packages/jQuery
2.  Tab (don't use the mouse) to the Version checkbox and press the space bar to select.

Result:  You'll lose control focus without warning.  This is a violation of WCAG 2.0 Level 'A'.
Expected:  Generally speaking, we shouldn't be initiating changes of context in places where users won't expect it.  In this case, focus should stay on the checkbox when you alter its value.

**Comment by johnataylor** [comment]
Site issues are being addressed as part of the v3 gallery development
